frontend/components/maps/MapMicro.js

```diff
@@ -376,6 +377,10 @@ export default class MapMicro extends React.Component {
                             url="http://stamen-tiles-a.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}.png"
                         />
                         {markerObjects}
+                        <div style={{marginTop: "17.5rem", marginLeft: "0.75rem",
```

Comment:
Please remove this from MapMicro.js! I think you are currently working in MapLegend.js